devops/scripts/benchmarks/benches/compute.py

```diff
@@ -631,6 +638,13 @@ def __init__(
             bench, f"multithread_benchmark_{self.runtime.value}", "MemcpyExecute"
         )
 
+    def enabled(self) -> bool:
+        if not super().enabled():
+            return False
+        if self.runtime == RUNTIMES.UR and options.ur is None:
```

Comment:
if self.runtime not in self.enabled_runtimes(
            [ RUNTIMES.UR]
        ):